src/main/java/io/coala/jetbrains/utils/CodeAnalysisRunner.java

```diff
@@ -0,0 +1,55 @@
+package io.coala.jetbrains.utils;
+
+import com.intellij.execution.ExecutionException;
+import com.intellij.execution.configurations.GeneralCommandLine;
+import com.intellij.execution.process.OSProcessHandler;
+
+import javax.annotation.Nonnull;
+
+public class CodeAnalysisRunner {
+    /**
+     * This method creates a runnable process and runs it.
+     * {@link OSProcessHandler#startNotify()} is used to capture the output
+     *
+     * @param cwd the current working directory for coala
+     * @param executable the path to coala executable
+     * @param section the coala section tag to run analysis on
+     * @throws ExecutionException
+     */
+    public static void submit(@Nonnull String cwd, @Nonnull String executable, @Nonnull String section) throws ExecutionException {
```

Comment:
Line is too long, are we following any guideline?

Reply:
Not enforcing any particular guideline atm. Following some basic Oracle and Google Java Guidelines (and adhering to default IntelliJ in editor warnings/errors) and trying to maintain consistency.